Fixing formatting on files.

mikeodr opened the pull request:
Astyle fixup.


Ran the following in the repo:
```bash
find -regextype egrep -regex '.*\.[ch](pp)?$' -exec astyle '{}' --style=allman --indent=spaces=2 --pad-oper --unpad-paren --pad-header --convert-tabs \;
```

mikepurvis commented:
I was picturing just doing `rosserial_client`— my main worry about the other repos is that there's some vendor code in there that probably shouldn't be restyled.

That said, maybe it does make sense to just fix everything?

mikeodr commented:
Will pair it back then.

Commit:
Fixing formatting on files.

find -regextype egrep -regex '.*\.[ch](pp)?$' -exec astyle '{}' --style=allman --indent=spaces=2 --pad-oper
--unpad-paren --pad-header --convert-tabs \;

mikepurvis commented:
Thanks, that is much more manageable.
